torch/_refs/__init__.py

```diff
@@ -628,10 +628,15 @@ def _sigmoid(a: TensorLikeType) -> TensorLikeType:
     type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.INT_TO_FLOAT,
 )
 
+
+def _square(a: TensorLikeType) -> TensorLikeType:
+    return mul(a, a)
+
+
 square = _make_elementwise_unary_reference(
-    prims.square,
+    _square,
     type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.BOOL_TO_LONG,
-    aten_op=None,  # CompositeImplicitAutograd,
+    aten_op=torch.ops.aten.square,
```

Comment:
Nice

Reply:
don't do this, this is why the tests are failing

Reply:
Thanks!